test/array-analysis.test.js

```diff
@@ -0,0 +1,34 @@
+const analyze = require('../src/array-analysis');
+
+const array = [1, 8, 3, 4, 2, 6];
+const obj = analyze(array);
+
+test('it analyses an array and returns an object', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6])).toStrictEqual({
+    average: 4, min: 1, max: 8, length: 6,
+  });
+});
+
+test('object returned has an average property', () => {
+  expect(obj.average).toBe(4);
+});
+
+test('object returned has an min property', () => {
+  expect(obj.min).toBe(1);
+});
+
+test('object returned has an max property', () => {
+  expect(obj.max).toBe(8);
+});
+
+test('object returned has an length property', () => {
+  expect(obj.length).toBe(6);
+});
```

Comment:
These tests might be redundant, you are already checking this with the first one. Maybe you can include some kind of other edge cases instead of double-checking here, for example: [1, null], [0], [1,1,1,1], [-Infinity, Infinity], etc.